rollup.config.js

```diff
@@ -0,0 +1,73 @@
+import resolve from '@rollup/plugin-node-resolve';
+import {terser} from 'rollup-plugin-terser';
+import {name, version, homepage, main} from './package.json';
+
+const banner = `/*!
+* ${name} v${version}
+* ${homepage}
+ * (c) ${(new Date(process.env.SOURCE_DATE_EPOCH ? (process.env.SOURCE_DATE_EPOCH * 1000) : new Date().getTime())).getFullYear()} Chart.js Contributors
```

Comment:
Should we change `Chart.js Contributors` to `chartjs-plugin-annotation Contributors`?